src/core_modules/capture-core/components/FiltersForTypes/Date/DateFilter.component.js

```diff
@@ -1,6 +1,7 @@
 // @flow
```

Comment:
The date working list filter label is in Gregorian format.
<img width="431" alt="Screenshot 2024-12-10 at 11 55 31" src="https://github.com/user-attachments/assets/c0db5b76-188e-4bba-b2b1-2806675ebcf0">